android-record/src/moditect/module-info.java

```diff
@@ -0,0 +1,9 @@
+module com.fasterxml.jackson.module.androidrecord {
+
+    requires com.fasterxml.jackson.databind;
```

Comment:
Jackson core and annotations here as well, I think.

Reply:
Done. But see [question above](https://github.com/FasterXML/jackson-modules-base/pull/227#discussion_r1380933330).

Reply:
Ok. I think databind+annotations or databind+core+annotations both fine, just prefer uses same set in pom and module-info.

Reply:
Ok, done.